### ISSUE TITLE
`UndefinedObject does not understand #asString` from `Browse Selected method` in SUnit Browser

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#828]
All three selected methods exist (seems I cannot browse any of the selected  methods in SUnit Browser (1 or many selections) right now) ... project browse is open on one of the methods so they all exist  :
![image](https://user-images.githubusercontent.com/623951/85806545-29f69700-b704-11ea-9d03-55ab94848ed0.png)

```
************************** Dolphin Virtual Machine Dump Report ***************************

16:47:42 PM, 6/25/2020: UndefinedObject does not understand #asString

*----> VM Context <----*
Process: {20A50210:size 402 words, suspended frame 20A50591, priority 5, callbacks 1, FP control 80007, thread 00000000}
Active Method: SessionManager>>logError:
IP: 00AE5149 (9)
SP: 20A5095C
BP: 20A50930 (441)
ActiveFrame: {20A50934: cf 20A50915, sp 20A5094C, bp 20A50930, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood


New Method: VMLibrary>>dump:path:stackDepth:walkbackDepth:
Message Selector: dump:path:stackDepth:walkbackDepth:

*----> Stack Back Trace <----*
{20A50934: cf 20A50915, sp 20A5094C, bp 20A50930, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood

{20A50914: cf 20A508F1, sp 20A50928, bp 20A5090C, ip 39, JadeSessionManager>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood
	stack temp[0]: 8'16:47:42 6/25/2020'
	env temp[0]: a FileStream

{20A508F0: cf 20A508D1, sp 20A50904, bp 20A508EC, ip 3, JadeSessionManager(SessionManager)>>unhandledException:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood

{20A508D0: cf 20A508B1, sp 20A508E4, bp 20A508CC, ip 3, JadeSessionManager(SessionManager)>>onUnhandledError:}
	receiver: a JadeSessionManager
	arg[0]: a MessageNotUnderstood

{20A508B0: cf 20A50895, sp 20A508C4, bp 20A508B0, ip 5, MessageNotUnderstood(Error)>>defaultAction}
	receiver: a MessageNotUnderstood

{20A50894: cf 20A50861, sp 20A508A8, bp 20A5087C, ip 55, MessageNotUnderstood(Exception)>>_propagateFrom:}
	receiver: a MessageNotUnderstood
	arg[0]: a ExceptionHandler
	stack temp[0]: nil
	stack temp[1]: a ExceptionHandler
	stack temp[2]: nil
	stack temp[3]: a Process('Main' base 20A50000* in SessionManager>>logError: sp=00000000 ip=4 list=02AD0010)
	stack temp[4]: nil

{20A50860: cf 20A50841, sp 20A50874, bp 20A5085C, ip 6, MessageNotUnderstood(Exception)>>_propagate}
	receiver: a MessageNotUnderstood
	stack temp[0]: nil

{20A50840: cf 20A50825, sp 20A50854, bp 20A50840, ip 13, MessageNotUnderstood(Exception)>>signal}
	receiver: a MessageNotUnderstood

{20A50824: cf 20A50801, sp 20A50838, bp 20A5081C, ip 12, MessageNotUnderstood class>>receiver:message:}
	receiver: MessageNotUnderstood
	arg[0]: nil
	arg[1]: Message selector: asString arguments: 03323DF0

{20A50800: cf 20A507E1, sp 20A50814, bp 20A507FC, ip 4, Object>>doesNotUnderstand:}
	receiver: nil
	arg[0]: Message selector: asString arguments: 03323DF0

{20A507E0: cf 20A507C1, sp 20A507F4, bp 20A507DC, ip 3, RowanMethodService>>className:}
	receiver: a RowanMethodService
	arg[0]: nil

{20A507C0: cf 20A50799, sp 20A507D4, bp 20A507BC, ip 10, [] in JadeiteSUnitAbstractPresenter>>browseSelectedMethods}
	receiver: nil
	arg[0]: a RowanMethodService

{20A50798: cf 20A50765, sp 20A507B4, bp 20A50788, ip 19, [] in SequenceableCollection>>collect:}
	receiver: nil
	arg[0]: 1
	arg[1]: a RowanMethodService

{20A50764: cf 20A50739, sp 20A50780, bp 20A50754, ip 13, Array(SequenceableCollection)>>uncheckedFrom:to:keysAndValuesDo:}
	receiver: a Array
	arg[0]: 1
	arg[1]: 3
	arg[2]: [] @ 14 in SequenceableCollection>>collect:
	stack temp[0]: 1

{20A50738: cf 20A50719, sp 20A5074C, bp 20A50734, ip 8, Array(SequenceableCollection)>>keysAndValuesDo:}
	receiver: a Array
	arg[0]: [] @ 14 in SequenceableCollection>>collect:

{20A50718: cf 20A506F5, sp 20A5072C, bp 20A50710, ip 22, Array(SequenceableCollection)>>collect:}
	receiver: a Array
	arg[0]: [] @ 5 in JadeiteSUnitAbstractPresenter>>browseSelectedMethods
	stack temp[0]: a Array

{20A506F4: cf 20A506D1, sp 20A50708, bp 20A506EC, ip 12, JadeiteSUnitPresenter(JadeiteSUnitAbstractPresenter)>>browseSelectedMethods}
	receiver: a JadeiteSUnitPresenter
	stack temp[0]: nil
	stack temp[1]: nil

{20A506D0: cf 20A506B5, sp 20A506E4, bp 20A506D0, ip 2, JadeiteSUnitPresenter>>browseMethods}
	receiver: a JadeiteSUnitPresenter

{20A506B4: cf 20A50695, sp 20A506C8, bp 20A506B0, ip 4, Symbol>>forwardTo:}
	receiver: browseMethods
	arg[0]: a JadeiteSUnitPresenter

{20A50694: cf 20A50675, sp 20A506A8, bp 20A50690, ip 4, CommandDescription>>performAgainst:}
	receiver: a CommandDescription
	arg[0]: a JadeiteSUnitPresenter

{20A50674: cf 20A50651, sp 20A50688, bp 20A50674, ip 16, [] in Command>>value}
	receiver: a Command

{20A50650: cf 20A50631, sp 20A50664, bp 20A5064C, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 25 in Command>>value

{20A50630: cf 20A5060D, sp 20A50644, bp 20A50628, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 13 in Command>>value
	arg[0]: [] @ 25 in Command>>value
	stack temp[0]: nil

{20A5060C: cf 20A505ED, sp 20A50620, bp 20A50608, ip 30, Command>>value}
	receiver: a Command
	stack temp[0]: nil

{20A505EC: cf 20A505CD, sp 20A50600, bp 20A505E8, ip 30, ShellView>>performCommand:}
	receiver: a ShellView
	arg[0]: a Command

{20A505CC: cf 20A505AD, sp 20A505E0, bp 20A505C8, ip 4, JadeiteSUnitBrowser(Shell)>>performCommand:}
	receiver: a JadeiteSUnitBrowser
	arg[0]: a Command

{20A505AC: cf 20A50585, sp 20A505C0, bp 20A505A0, ip 10, CommandQuery>>perform}
	receiver: a CommandQuery
	stack temp[0]: a Command
	stack temp[1]: nil
	stack temp[2]: a JadeiteSUnitBrowser

{20A50584: cf 20A50561, sp 20A50598, bp 20A5057C, ip 8, DelegatingCommandPolicy(CommandPolicy)>>route:}
	receiver: a DelegatingCommandPolicy
	arg[0]: a CommandDescription
	stack temp[0]: a CommandQuery

{20A50560: cf 20A50539, sp 20A50574, bp 20A5055C, ip 15, [] in MultipleSelectionListView(View)>>onCommand:}
	receiver: a MultipleSelectionListView

{20A50538: cf 20A50519, sp 20A5054C, bp 20A50534, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{20A50518: cf 20A504F5, sp 20A5052C, bp 20A50510, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in View>>onCommand:
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{20A504F4: cf 20A504CD, sp 20A50508, bp 20A504E8, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 11 in View>>onCommand:
	stack temp[0]: nil
	stack temp[1]: a ExternalHandle(00010003)

{20A504CC: cf 20A504A9, sp 20A504E0, bp 20A504C4, ip 17, MultipleSelectionListView(View)>>onCommand:}
	receiver: a MultipleSelectionListView
	arg[0]: a CommandDescription
	stack temp[0]: nil

{20A504A8: cf 20A50475, sp 20A504BC, bp 20A50490, ip 19, MultipleSelectionListView(View)>>wmCommand:wParam:lParam:}
	receiver: a MultipleSelectionListView
	arg[0]: 273
	arg[1]: 29647
	arg[2]: 0
	stack temp[0]: 29647
	stack temp[1]: a CommandDescription
	stack temp[2]: nil

{20A50474: cf 20A50445, sp 20A50488, bp 20A50460, ip 23, MultipleSelectionListView(View)>>dispatchMessage:wParam:lParam:}
	receiver: a MultipleSelectionListView
	arg[0]: 273
	arg[1]: 29647
	arg[2]: 0
	stack temp[0]: nil
	stack temp[1]: wmCommand:wParam:lParam:

{20A50444: cf 20A50411, sp 20A50458, bp 20A50434, ip 69, [] in InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: nil

{20A50410: cf 20A503E9, sp 20A50424, bp 20A5040C, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 79 in InputState>>wndProc:message:wParam:lParam:cookie:

{20A503E8: cf 20A503AD, sp 20A50404, bp 20A503D0, ip 84, GuiInputState(InputState)>>wndProc:message:wParam:lParam:cookie:}
	receiver: a GuiInputState
	arg[0]: 919224
	arg[1]: 273
	arg[2]: 29647
	arg[3]: 0
	arg[4]: 851152
	stack temp[0]: a MultipleSelectionListView

{20A503AC: cf 20A5038D, sp 20A503C8, bp 20A503A8, ip 28, GuiInputState(InputState)>>pumpMessage:}
	receiver: a GuiInputState
	arg[0]: a MSG

{20A5038C: cf 20A50361, sp 20A503A0, bp 20A5037C, ip 18, GuiInputState(InputState)>>loopWhile:}
	receiver: a GuiInputState
	arg[0]: [] @ 9 in InputState>>mainLoop
	stack temp[0]: a MSG
	stack temp[1]: true
	stack temp[2]: [] @ 8 in JadeiteMethodSourcePresenter>>updateMethodSourceColor

{20A50360: cf 20A50345, sp 20A50374, bp 20A50360, ip 15, GuiInputState(InputState)>>mainLoop}
	receiver: a GuiInputState

{20A50344: cf 20A50329, sp 20A50358, bp 20A50344, ip 14, [] in GuiInputState(InputState)>>forkMain}
	receiver: a GuiInputState

{20A50328: cf 20A5030D, sp 20A5033C, bp 20A50328, ip 13, ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry}
	receiver: a ExceptionHandler

{20A5030C: cf 20A502E5, sp 20A50320, bp 20A50308, ip 22, [] in ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler

{20A502E4: cf 20A502C5, sp 20A502F8, bp 20A502E0, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:

{20A502C4: cf 20A502A1, sp 20A502D8, bp 20A502BC, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 17 in ExceptionHandlerAbstract>>try:
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:
	stack temp[0]: nil

{20A502A0: cf 20A50279, sp 20A502B4, bp 20A50294, ip 44, ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler
	arg[0]: [] @ 10 in InputState>>forkMain
	stack temp[0]: nil
	stack temp[1]: a Process('Main' base 20A50000* in SessionManager>>logError: sp=00000000 ip=4 list=02AD0010)
	env temp[0]: nil

{20A50278: cf 20A50255, sp 20A5028C, bp 20A50270, ip 7, BlockClosure>>on:do:}
	receiver: [] @ 10 in InputState>>forkMain
	arg[0]: ProcessTermination
	arg[1]: [] @ 13 in BlockClosure>>newProcess:

{20A50254: cf 00000001, sp 20A50268, bp 20A50254, ip 17, [] in BlockClosure>>newProcess:}
	receiver: [] @ 10 in InputState>>forkMain

<Bottom of stack>

***** End of dump *****

***** Ending Dolphin stack dump started at 16:47:42 6/25/2020 *****
```

**Comment by dalehenrich** [comment]
Browsed all three selected methods from the project browser without error .... still can't browse from SUnit Browser

**Comment by ericwinger** [comment]
Fixed in Jadeite 3.2. When the sunit browser was open on a test class when a new test was added in another browser, the method service was not getting the `definedClassName` inst var updated correctly. Without that, the test doesn't know which method to browse when `Browse Selected Tests` is selected. Fix in `Rowan`. Tests in both `Rowan` and `Jadeite` projects.
